package.json

```diff
@@ -6,7 +6,9 @@
 	"license": "GPL-2.0-or-later",
 	"private": true,
 	"dependencies": {
-		"@wordpress/env": "4.1.3"
+		"@wordpress/env": "9.3.0",
+		"@wordpress/scripts": "27.2.0",
+		"@wordpress/stylelint-config": "21.34.0"
```

Comment:
Dependencies updated to match the [Pattern Directory theme](https://github.com/WordPress/pattern-directory/blob/d895a721a9390f35d67982814e4a4030ad5a88f1/package.json#L8)